app/models.py

```diff
@@ -2731,8 +2731,9 @@ class Report(BaseModel):
         nullable=False,
     )
     job_id = db.Column(UUID(as_uuid=True), db.ForeignKey("jobs.id"), nullable=True)  # only set if report is for a bulk job
-    url = db.Column(db.String(255), nullable=True)  # url to download the report from s3
+    url = db.Column(db.String(800), nullable=True)  # url to download the report from s3
```

Comment:
this is a new table right and has minimal data? else changing the column type is a blocking transaction and can take long.

Reply:
44 is very small. you should be fine